src/server.ts

```
@@ -55,6 +55,9 @@ app.post('/', async (request, reply) => {
       return;
     case 'health_status':
       return reply.code(200).send({result: 'ok'});
+    case 'sign':
+      const signature = await kmsProvider.signDigest(keyId, Buffer.from(params[0], 'hex'));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)